src/Popover.vue

```diff
@@ -179,6 +179,10 @@ export default {
 <style lang="scss">
 $pointer-size: 6px;
 
+popover {
```

Comment:
What is this? You cannot style "components" directly and you do not render <popover/>